src/main/resources/static/auth/login.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+<script>
+        const urlParams = new URLSearchParams(window.location.search);
+        const token = urlParams.get('token');
+        localStorage.setItem('token', token);
+        window.location.replace("http://localhost:8080")
```

Comment:
here `window.location.replace`
in some other place`location.href=` 
pls explain difference

Reply:
ok, nice :D